### DISCUSSION TITLE
Android explore tab scrolling janky

### DISCUSSION THREAD

**Discussion by Alizter** [community/community#39415] [Mobile]
When scrolling int he explore tab on android the scrolling jumps and is not smooth.

**Comment by aconsuegra** [comment]
Hi @Alizter, thanks for reaching out with your feedback. Could you share the version of the app, device model and Android version that you are using?

**Reply by Alizter** [reply]
App Version 1.89.0
Android: 12
Phone: Galaxy A51

**Reply by Alizter** [reply]
This is still happening 2 months later BTW.
